### ISSUE TITLE
[Bug] storybook no longer works with react-native-reanimated

### ISSUE THREAD

**Issue by ppong** [storybookjs/addon-react-native-web#61]
### Describe the bug

`react-native-reanimated` doesn't work with latest `@storybook/addon-react-native-web` and throw the following error. I tried debugging this for a while but I am kind of out of ideas on what else to try.

```
caught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at _updatePropsJS (index.js:1:1)
    at UpdateProps.js:1:1
    at Set.forEach (<anonymous>)
    at _f (UpdateProps.js:1:1)
    at _f (useAnimatedStyle.js:2:1)
    at Mapper._f [as mapper] (useAnimatedStyle.js:2:1)
    at Mapper.execute (Mapper.js:1:1)
    at MapperRegistry.execute (MapperRegistry.js:1:1)
    at JSReanimated._onRender (JSReanimated.js:1:1)
_
```

### Steps to reproduce the behavior

I created a minimal example by running
```
npx sb init --type react
yarn add react-dom react-native-web babel-plugin-react-native-web @storybook/addon-react-native-web --dev
```
and also updated the config
```
module.exports = {
  addons: [
    /*existing addons,*/
    {
      name: '@storybook/addon-react-native-web',
      options: {
        modulesToTranspile: ['react-native-reanimated'],
        babelPlugins: ['react-native-reanimated/plugin'],
      },
    },
  ],
};
```

Here is a repo with the minimal example: https://github.com/ppong/addon-react-native-web-debug

### Screenshots and/or logs

<img width="2095" alt="Screenshot 2023-04-25 at 7 53 36 PM" src="https://user-images.githubusercontent.com/1063140/234432484-f5676236-aa11-4442-8a39-f2a4c3415dcd.png">

<img width="1163" alt="Screenshot 2023-04-25 at 7 59 24 PM" src="https://user-images.githubusercontent.com/1063140/234432990-2b2494b7-8b9b-4eb1-bf34-560bbeb67dac.png">

### Environment

 - OS: MacOS
 - Node.js version: v18.15.0
 - NPM version: 9.5.0
 - Browser (if applicable): chrome

**Comment by dannyhw** [comment]
Looks like you're missing the namespace plugin

`@babel/plugin-proposal-export-namespace-from',`

See here https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/web-support

**Comment by ppong** [comment]
> Looks like you're missing the namespace plugin
> 
> `@babel/plugin-proposal-export-namespace-from',`
> 
> See here https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/web-support

I actually tried this previously but got this error. I think `@babel/plugin-proposal-export-namespace-from',` was already included

```
ModuleBuildError: Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Duplicate plugin/preset detected.
If you'd like to use two separate instances of a plugin,
they need separate names, e.g.

  plugins: [
    ['some-plugin', {}],
    ['some-plugin', {}, 'some unique name'],
  ]

Duplicates detected are:
[
  {
    "alias": "/Users/work/Documents/sb-reanimated/node_modules/@babel/plugin-proposal-export-namespace-from/lib/index.js",
    "dirname": "/Users/work/Documents/sb-reanimated",
    "ownPass": false,
    "file": {
      "request": "@babel/plugin-proposal-export-namespace-from",
      "resolved": "/Users/work/Documents/sb-reanimated/node_modules/@babel/plugin-proposal-export-namespace-from/lib/index.js"
    }
  },
  {
    "alias": "/Users/work/Documents/sb-reanimated/node_modules/@babel/plugin-proposal-export-namespace-from/lib/index.js",
    "dirname": "/Users/work/Documents/sb-reanimated",
    "ownPass": false,
    "file": {
      "request": "@babel/plugin-proposal-export-namespace-from",
      "resolved": "/Users/work/Documents/sb-reanimated/node_modules/@babel/plugin-proposal-export-namespace-from/lib/index.js"
    }
  }
]```

**Comment by dannyhw** [comment]
in v7 you need to have these things in your project babel config, for example here:

https://github.com/ppong/addon-react-native-web-debug/blob/main/.babelrc.json

**Comment by ppong** [comment]
@dannyhw really appreciate the help

**Comment by ppong** [comment]
hi @dannyhw just wondering if you had a chance to take a look? I tried to mess with the `babelrc.json` but didn't have any luck getting it to work. Would you recommend downgrading to storybook v6.5?

**Comment by ppong** [comment]
closing this: seems to be fixed with storybook 7.0.9

**Comment by dannyhw** [comment]
@ppong hey sorry that I didn't respond before been super busy, glad to hear it got resolved :)
